clang/test/Driver/hlsl-lang-targets-spirv.hlsl

```diff
@@ -2,8 +2,8 @@
 
 // Supported targets
 //
-// RUN: %clang -target dxil-unknown-shadermodel6.2-pixel %s -S -o /dev/null 2>&1 | FileCheck --check-prefix=CHECK-VALID %s
-// RUN: %clang -target spirv-unknown-shadermodel6.2-library %s -S -o /dev/null 2>&1 | FileCheck --check-prefix=CHECK-VALID %s
+// RUN: %clang -target dxil-unknown-shadermodel6.2-compute %s -S -o /dev/null 2>&1 | FileCheck --allow-empty --check-prefix=CHECK-VALID %s
```

Comment:
`FileCheck` fails by default when the input is empty, which is the case when this test succeeds (the actual compilation output is being sent to `/dev/null`, just stdout/stderr messages are checked). The dxil case was succeeding because there's always the `clang: warning: dxv not found. Resulting DXIL will not be validated or signed for use in release environments.` message, but since that's not the thing this test is checking I figured it made more sense to `--allow-empty` for both.

Reply:
Ahh, thanks for the explanation. Yes, sounds better not to rely on an unchecked warning! Thanks!